package.json

```diff
@@ -6,7 +6,7 @@
   "scripts": {
     "test": "jest --selectProjects=server --selectProjects=shared --verbose",
     "integration-server": "git submodule update --init --recursive && shx cp test-harness/features/evaluation.feature packages/server/integration/features && jest --selectProjects=server-e2e --verbose",
-    "integration-client": "git submodule update --init --recursive && shx cp test-harness/features/evaluation.feature packages/client/integration/features && jest --selectProjects=client-e2e --verbose",
+    "integration-client": "jest --selectProjects=client-e2e --verbose",
```

Comment:
I've temporarily removed the module pull and copy so you can add tests 1 by 1. Currently only the first test is implemented.